gui-tk/minsky.tcl

```diff
@@ -273,7 +273,7 @@ if {[tk windowingsystem] == "aqua"} {
     event add <<middleMouse-Motion>> <B3-Motion>
     event add <<middleMouse-ButtonRelease>> <B3-ButtonRelease>
 } else {
-    event add <<contextMenu>> <Button-3>
+    event add <<contextMenu>> <Button-3> <Control-Button-2>
```

Comment:
Yes that should work, except it should be <Control-Button-1>, not <Control-Button-2>
Very neat - even I didn't think it would be quite this simple.

Reply:
Thanks, I've fixed it :)

Reply:
I'm not sure it works... I don't know if it is a limitation of Ubuntu, but it doesn't work on my system.

Reply:
I've been reading the documentation and it seems to me that the second binding <Control-Button-1> is being ignored (see under the heading "Multiple Matches":

[https://www.tcl.tk/man/tcl8.4/TkCmd/bind.htm](https://www.tcl.tk/man/tcl8.4/TkCmd/bind.htm)

As far as I understand second bindings get ignored when they have been attached to some other event or binding, but upon doing a "grep -r" on <Control-Button-1> (or <Meta-Button-1> or other variations thereof) from Minsky's top directory, I don't see it anywhere else but inside minsky.tcl where I've made the change.

Maybe it is my Spanish keyboard? I say this because I disabled the right-click button event and only left <Control-Button-1> active, and it still did nothing when I tried it on the Canvas. Would you mind trying it out on yours, please? Thanks.

Reply:
Control double left mouse click works. But why doesn't control left mouse click?

Reply:
I think I've worked out the issue. Control mouse is already bound to individually adding and removing items from a selection. Could you do a bit more research as to how to deal with context menus in that case? Control mouse click is a common idion for individual selections, we don't want to lose that.

Reply:
Ah, I didn't see the "Control-ButtonPRESS-1" bound to controlMouseDown. Thanks for pointing it out, I was only searching for "Control-Button-1". I've tried "Alt-Button-1", but I think it has already been bound to mouse coordinates. Then, I've managed to check  that "Control-Double-Button-1" works like you mentioned at sourceforge. I will see if I can't use the [bindtags](https://www.tcl.tk/man/tcl8.4/TkCmd/bindtags.htm) command to change the order in which "Control-Button-1" is executed. Since in once case it is used to select an item and the other to open the context menu on the background canvas, it may be possible to differentiate between those two events using bindtags.